Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "LoggingPolyglot"
 uuid = "211639cc-9b11-4cfd-abc6-8f7477829344"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
I did something not very smart and committed 0.3.0 in master. This will lead to a conflict but we should release as 0.3.0 since there are new features.